src/test/java/org/sagebionetworks/web/unitclient/widget/profile/UserProfileEditorWidgetImplTest.java

```diff
@@ -203,6 +203,7 @@ private void verifyConfigure(int nCalls) {
     verify(mockSynAlert, times(nCalls)).clear();
     verify(mockView, times(nCalls)).hideLinkError();
     verify(mockView, times(nCalls)).hideUsernameError();
+    verify(mockView, times(nCalls)).clearEmails();
```

Comment:
Add a new clearEmails() method, and verify that it is called whenever the profile editor is configured